checker/tests/guieffect/ThrowCatchTest.java

```diff
@@ -17,15 +17,15 @@ class Inner {}
     // Type var test
     <E extends @UI PolyUIException> void throwTypeVarUI1(E ex1, @UI E ex2) throws PolyUIException {
         if (flag) {
-            // :: error: (throw.type.invalid)
+            // :: error: (throw.type.incompatible)
```

Comment:
Changing the error key also requires a change to this file:
https://github.com/eisop/checker-framework/blob/e604d8d10a38d09f7c3e94c4d614b2d729f48085/framework/src/main/java/org/checkerframework/common/basetype/messages.properties#L35C12-L35C19

`incompatible` is the better term, as there is a comparison between two types.
In a [previous discussion](https://github.com/eisop/checker-framework/pull/436#discussion_r1342004418) we said we do this renaming in a separate PR. Was there a change of mind?

Reply:
I have opened a new PR here #618.

Reply:
So you should undo these changes in this PR, so that we can merge this PR independently of #618.